Annotation/constituent_golden_set.json

```diff
@@ -334,7 +341,19 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "why the Conservancy is stepping up its collaboration with IHA",
+                "embedded clauses": []
+            },
+            {
+                "predicate": [
+                    {
+                        "str": "said",
+                        "lemma": "",
+                        "POS": ""
+                    }
+                ],
+                "type": "declarative",

```

Comment:
Maybe not an embedded clause because it's a direct quote (so not really embedded?)